src/ADAL.PCL.iOS/TokenCachePlugin.cs

```diff
@@ -26,12 +29,66 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
     internal class TokenCachePlugin : ITokenCachePlugin
     {
+        private const string LocalSettingsContainerName = "ActiveDirectoryAuthenticationLibrary";
```

Comment:
eventually, we will be same. It would be better to follow native's platform's naming convention.